docs/sql-programming-guide.md

```diff
@@ -1371,6 +1380,26 @@ Configuration of Parquet can be done using the `setConf` method on `SQLContext`
     </p>
   </td>
 </tr>
+<tr>
+  <td><code>spark.sql.parquet.mergeSchema</code></td>
+  <td><code>false</code></td>
+  <td>
+    <p>
+      When true, the Parquet data source merges schemas collected from all data files, otherwise the
+      schema is picked from the summary file or a random data file if no summary file is available.
+    </p>
+  </td>
+</tr>
+<tr>
+  <td><code>spark.sql.parquet.mergeSchema</code></td>
```

Comment:
CC @liancheng 
@witgo I think you could have just opened a PR to remove this duplicated content as an addendum to SPARK-9424. You opened SPARK-10350 / https://github.com/apache/spark/pull/8520 which is OK, but you didn't tie the two together anywhere. The title and description still don't make this clear.